src/Umbraco.Web.BackOffice/Controllers/MemberTypeController.cs

```diff
@@ -172,6 +173,23 @@ public IActionResult DeleteById(int id)
         return Ok(result);
     }
 
+    /// <summary>
+    ///     Returns where a particular composition has been used
+    ///     This has been wrapped in a dto instead of simple parameters to support having multiple parameters in post request
+    ///     body
+    /// </summary>
+    /// <param name="filter"></param>
+    /// <returns></returns>
+    [HttpPost]
+    [Authorize(Policy = AuthorizationPolicies.TreeAccessMediaTypes)]
```

Comment:
As above, adding this attribute is redundant as the whole controller already adheres to this policy.